src/vs/base/browser/ui/inputbox/inputBox.ts

```diff
@@ -364,7 +364,11 @@ export class InputBox extends Widget {
 				};
 			}
 
-			if (!errorMsg) {
+			if (errorMsg) {
+				this.inputElement.setAttribute('aria-invalid', 'true');
+				this.showMessage(errorMsg);
+			}
+			else if (this.inputElement.hasAttribute('aria-invalid')) {
```

Comment:
This code matches what's in VS Code's version of `inputBox.ts` now so it doesn't need "{{SQL CARBON EDIT}}" tags. The only reason it didn't match before is because I resolved a merge conflict (created because the block of code above this is actually something we edited) incorrectly.